merkletree/pad.go

```diff
@@ -10,7 +10,13 @@ import (
 )
 
 var (
+	// ErrorSTRNotFound indicates that the STR of the searching epoch
+	// has been removed from the memory, because the snapshot array has exceeded
+	// its maximum length.
 	ErrorSTRNotFound = errors.New("[merkletree] STR not found")
+
+	// ErrorNilPolicies is used to panic if the PAD was created
+	// with a nil Policies.
```

Comment:
I would change this to "a nil Polices struct". Otherwise, this is grammatically awkward even though it's correct terminology.